client/package.json

```diff
@@ -14,13 +16,16 @@
     "@testing-library/react": "^11.2.2",
     "@testing-library/user-event": "^12.2.2",
     "babel-loader": "^8.1.0",
+    "date-fns": "^2.16.1",
     "eslint": "^7.14.0",
     "eslint-plugin-react": "^7.21.5",
     "react": "^17.0.1",
     "react-datepicker": "^3.3.0",
     "react-dom": "^17.0.1",
+    "react-google-maps": "^9.4.5",
```

Comment:
Is this used? Consider removing it if it's unused.

Reply:
I uninstalled it. Thanks